src/core/reference.js

```diff
@@ -0,0 +1,344 @@
+/**
+ * @module Foundation
+ * @submodule Declaration
+ * @for p5
+ */
+
+
+/**
+
```

Comment:
The docs won't build correctly if there is a space like this here. There can be no empty lines between a `/**` and a `*/`